src/server/seeds/dev/006_00_vessels.js

```diff
@@ -12,17 +12,17 @@ exports.seed = function(knex, Promise) {
       return knex("vessels").insert([
         {
           organization_id: 1,
-          mmsi: 23.33,
+          mmsi: "whatever",
```

Comment:
i am not sure if it is really required to change the seed values as well?

Reply:
@fawadik as @zkwsk asked me before, whenever I have to do change the migrate file I have to make the proper change to the seeds.

Reply:
MMSI - Maritime Mobile Service Identity it’s the ID that is used for radio/frequency communications and it can be used as an unique identifier of a Vessel.